README.md

```diff
@@ -0,0 +1,64 @@
+<h1 align='center'> PS Back-end Mega 2023 </h1>
+
+<section align='center'>
+<img alt="Top language" src="https://img.shields.io/github/languages/top/falcao-g/processo-mega">
+<img alt="Version express" src="https://img.shields.io/badge/express--red.svg?logo=express">
```

Comment:
It would be nice if you add the version on the badges
like this:
/express-^4.18.2-red.svg?logo=express

Reply:
I added this and was unsure of what to add in the postgres one so i added the version of the pg package

Reply:
Let's use the latest postgres version which is 15